ops-bedrock/l1-entrypoint.sh

```diff
@@ -15,6 +15,9 @@ cd genesis
 npm install
 cd ..
 
+sed -i -e "s/INIT_HOLDER=\"0x04d63aBCd2b9b1baa327f2Dda0f873F197ccd186\"/INIT_HOLDER=\"$INIT_HOLDER\"/g" .env
+sed -i -e "s/INIT_HOLDER_PRV=\"59ba8068eb256d520179e903f43dacf6d8d57d72bd306e1bd603fdb8c8da10e8\"/INIT_HOLDER_PRV=\"$INIT_HOLDER_PRV\"/g" .env
```

Comment:
Why is this necessary?  It seems `ops-bedrock/l1.env` already changed this

Reply:
l1.env just injects the value into the environment variable, but in the node-deploy repo of bsc, it will use its own .env to overwrite the environment variable, causing the modification of l1.env to become invalid.